content/blog/2016/2016-05-10-jenkins-20-vjam.adoc

```diff
@@ -0,0 +1,365 @@
+---
+layout: post
+author: rtyler
+title: "Jenkins 2.0 Online JAM Wrap-up"
+tags:
+- jenkins2
+- jam
+- meetup
+---
+
+
+Last week we hosted our first ever
+link:http://www.meetup.com/Jenkins-online-meetup/[Online JAM] with the debut
+topic of: link:/2.0/[Jenkins 2.0]. link:https://github.com/alyssat[Alyssa], our
+Events officer, and I pulled together a
+link:http://www.meetup.com/Jenkins-online-meetup/events/230154841/[series of
+sessions] focusing on some of the most notable aspects of Jenkins 2 with:
+
+* A Jenkins 2.0 keynote from project founder
+ link:https://github.com/kohsuke[Kohsuke Kawaguchi]
+* An overview of "Pipeline as Code" from link:https://github.com/hrmpw[Patrick
+  Wolf]
+* A deep-dive into Pipeline and related plugins like Multibranch, etc from
+  link:https://github.com/jglick[Jesse Glick] and
+  link:https://github.com/kishorebhatia[Kishore Bhatia]
+* An overview of new user experience changes in 2.0 from
+  link:https://github.com/kzantow[Keith Zantow]
+* A quick lightning talk about documentation by yours truly
+* Wrapping up the sessions, was Kohsuke again, talking about the road beyond
+  Jenkins 2.0 and what big projects he sees on the horizon.
+
+++++
+<center>
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/emV60CcDVV0" frameborder="0" allowfullscreen></iframe>
+</center>
+++++
+
+
+The event was really interesting for me, and I hope informative for those who
+participated in the live stream and Q&A session. I look forward to hosting more
+Virtual JAM events in the future, and I hope you will
+*link:http://www.meetup.com/Jenkins-online-meetup/[join us!]*
+
+=== Questions and Answers
+
+Below are a collection of questions and answers, that were posed during the
+Virtual JAM. Many of these were answered during the course of the sessions, but
+for posterity all are included below.
+
```

Comment:
Recording of the event could be found [HERE]( https://plus.google.com/events/cd3i7445t7819a1j2t0nboi1168)